source/integrate_with_govuk_pay/index.html.md.erb

```diff
@@ -0,0 +1,170 @@
+report on payments. You can read about the API in the [__API
+reference__](/api_reference) section of this documentation and use the
+interactive [API explorer](https://gds-payments.gelato.io/api-explorer/)
+[external link]. You can also read more about [__Reporting__](/reporting).
+
+## Mop-up jobs
+
+A mop-up job is a background process which checks the outcome of incomplete
+payment journeys.
+
+Most services need to release their service to the user automatically, upon
+successful completion of a payment. If you do this then you should plan for
+inevitable failures associated with unreliable internet connections and user
+behaviour. For example:
+
+* user abandons their payment journey before completing it
+* user completes their payment successfully, but their network connection is
+interrupted before they return to your service
+
+In these cases, users will never visit the `return_url` on your service. You
+should therefore not rely on the `return_url` to determine the outcome of the
+payment journey.
+
+### Using a mop-up job
+
+This section describes the recommended approach for most integrations. It
+provides the best guarantee that edge cases are handled properly and
+automatically.
+
+In order to use a mop-up job, you should have a datastore which keeps track of
+incomplete payment journeys.
+
+The following diagram gives a detailed overview of the sequence:
+
+![](images/keep-track-of-incomplete-payment-journeys.png)
+
+The following diagram shows an example of an incomplete payment journey, and
+how a mop-up job would clean it up:
+
+![](images/incomplete-payment-journeys-mop-up-jobs.png)
+
+Your mop-up job may run frequently, for example every half an hour. It should
+ignore recent payments to avoid interfering with in-flight payments. If it
+finds ‘stale’ incomplete payment journeys then it’s likely that payment
+journey has been abandoned. The mop-up job should call the GOV.UK Pay API to
+determine the status of the payment.
+
+### Integrations without mop-up jobs
+
+You may have a very low-volume service where you have a manual process for
+checking that payments succeeded. For example, using the GOV.UK Pay admin
+tool. You may therefore choose not to use a mop-up job. In this case, you
+should make sure your service staff are trained to check the outcomes of
+payments in the GOV.UK Pay admin tool before releasing the service to the end
+user.
+
+If you do not use a mop-up job, you will only be able to determine the outcome
+of a payment journey when the user visits the `return_url` on your service.
+
+<br>
+
+<div class="govuk-warning-text">
+  <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+  <strong class="govuk-warning-text__text">
+    <span class="govuk-warning-text__assistive">Warning</span>
+There are scenarios in which the `return_url` will never be visited, so there
+may be a small number of payment journeys left in a limbo state. You would
+need to manually reconcile these.
```

Comment:
i'm a bit confused by this warning text 
1. does it apply to payments in the admin tool or API? 
2. can 'limbo state' be defined as to what it actually is (e.g. payments left as 'in progress' or 'created'/'started'/'submitted' - depending on whether the payment is being viewed in the admin tool or API)
if admin tool, then 'in progress' payments are transitioned to 'timed out' after 90 minutes. if a service is relying on the API, then indeed there wouldn't have been a status update.
3. what exactly do we mean by manually reconcile here? manually update the 'service backend'? why would it be manual?